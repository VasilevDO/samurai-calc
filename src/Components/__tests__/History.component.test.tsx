import {render} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import 'jest-styled-components';
import History from '../History.component';

describe('HistoryComponent', () => {
	const history = ['1', '2', '3'];

	test('should render correct element', () => {
		const {container} = render(<History history={history}/>);
		expect(container).toMatchSnapshot();
	});
});

