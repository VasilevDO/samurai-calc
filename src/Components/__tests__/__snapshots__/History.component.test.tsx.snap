// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryComponent should render correct element 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column-reverse;
  -ms-flex-direction: column-reverse;
  flex-direction: column-reverse;
  -webkit-flex-flow: column-reverse;
  -ms-flex-flow: column-reverse;
  flex-flow: column-reverse;
  height: 117px;
  font-size: 24px;
  line-height: 32px;
  font-weight: 500;
  overflow: hidden;
}

.c0 > p {
  text-align: right;
  overflow-wrap: anywhere;
}

<div>
  <div
    class="c0"
    data-testid="history"
  >
    <p>
      3
    </p>
    <p>
      2
    </p>
    <p>
      1
    </p>
  </div>
</div>
`;
